pipeline/api/cloud.py

```diff
@@ -55,9 +55,19 @@ class PipelineCloud:
     token: Optional[str]
     url: Optional[str]
 
-    def __init__(self, url: str = None, token: str = None) -> None:
+    def __init__(
+        self,
+        *,
+        url: str = None,
+        token: str = None,
+        timeout=60.0,
+        verbose=True,
+        auth_check=True,
```

Comment:
not sure what the intention of this is - don't think it's currently used anywhere?

Reply:
Correct, removed.